scss/components/_contact-form.scss

```diff
@@ -0,0 +1,52 @@
+.contact-form {
+  padding: u(2rem);
+
+  .contact-form__element {
+    padding: u(1rem 0);
+  }
+
+  textarea {
+    height: u(15rem);
+  }
+
+  .tt-input {
+    @include u-icon-bg($magnifying-glass-arrow, $gray-dark);
+    background-position: right 10px top 50%;
+  }
+
+  .hr--soft {
```

Comment:
Just kind of a minor nitpick, but I think the hr naming convention might make more sense to follow the color variables, so like `.hr` and `.hr--dark` (and `.hr--light` if using `$gray-light`). What do you think?